src/preferencesdialog.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>400</width>
-    <height>400</height>
+    <width>427</width>
+    <height>409</height>
    </rect>
```

Comment:
Please don't change the above sizes when you edit a ui file. I know that Designer did it, based on your Qt style and font sizes but please restore them after using it.

The real sizes will be set by Qt when the app is used and based on user's GUI settings.

Reply:
I'll take care about here and the next times :)